web/vital_records/models.py

```diff
@@ -177,24 +177,24 @@ class VitalRecordsRequest(models.Model):
     status = FSMField(default="initialized", choices=STATUS_CHOICES)
     fire = models.CharField(max_length=50, choices=FIRE_CHOICES)
     relationship = models.CharField(max_length=50, choices=RELATIONSHIP_CHOICES)
-    legal_attestation = models.CharField(max_length=100)
-    first_name = models.CharField(max_length=100)
-    middle_name = models.CharField(max_length=100, blank=True)
-    last_name = models.CharField(max_length=100)
+    legal_attestation = models.CharField(max_length=386)
+    first_name = models.CharField(max_length=128)
+    middle_name = models.CharField(max_length=128, blank=True)
+    last_name = models.CharField(max_length=128)
     county_of_birth = models.CharField(max_length=15, choices=COUNTY_CHOICES)
     date_of_birth = models.DateField(null=True)
-    parent_1_first_name = models.CharField(max_length=100)
-    parent_1_last_name = models.CharField(max_length=100)
-    parent_2_first_name = models.CharField(max_length=100, blank=True)
-    parent_2_last_name = models.CharField(max_length=100, blank=True)
+    parent_1_first_name = models.CharField(max_length=128)
+    parent_1_last_name = models.CharField(max_length=128)
+    parent_2_first_name = models.CharField(max_length=128, blank=True)
+    parent_2_last_name = models.CharField(max_length=128, blank=True)
     number_of_records = models.IntegerField(choices=NUMBER_CHOICES, null=True)
-    order_first_name = models.CharField(max_length=50)
-    order_last_name = models.CharField(max_length=50)
+    order_first_name = models.CharField(max_length=128)
+    order_last_name = models.CharField(max_length=128)
     address = models.CharField(max_length=50)
```

Comment:
I don't see an `address_2` field being added to the request? So it was added to the form, but it won't get saved to the DB right?

I don't think it is being added to the generated PDF package either right? I think it needs to be added to the `PackageTask`?

E.g.

```python
application = Application(
   package_id=request_id,
   WildfireName=request.fire.capitalize(),
   # other fields...
   RequestorMailingAddress=" ".join(_filter_empty((request.address, request.address_2))).
   # other fields...
)
```

Or am I missing something?

Reply:
Ugh. I think I am going cross-eyed from all the rebasing and re-migrating and force pushing. There totally was an address_2 there previously T_T.